client/js/color.js

```diff
@@ -64,4 +66,38 @@ export function contrastAnyColor(inputColor, intensity, direction) {
   const toHexString = (r, g, b) => '#' + r.toString(16).padStart(2, '0') + g.toString(16).padStart(2, '0') + b.toString(16).padStart(2, '0');
   return toHexString(...adjustColor(r, g, b, modifier));
   return defaultColor;
+}
+
+export function calcHue(startingColors) {
+  let hue = 0;
+  const hues = [];
+  for(const player in startingColors) {
+    const hex = startingColors[player];
```

Comment:
Should call toHex so that input colors don't need to be in `#rrggbb` format.

Reply:
Yes.  I'll make that change.  It wasn't necessary before, but when we allow designers to use an array, it is necessary.